ui/src/routes/_layout/organizations/$orgId/index.tsx

```diff
@@ -148,7 +132,7 @@ const OrganizationComponent = () => {
   const table = useReactTable({
     data: products?.data || [],
     columns,
-    pageCount: Math.ceil(products.pagination.totalCount / pageSize),
+    pageCount: Math.ceil(products?.pagination.totalCount ?? 0 / pageSize),
```

Comment:
As a follow-up I'd consider moving the table to a separate component, so the table hook is only executed when the data is available.

Reply:
Could this be done in a follow-up PR? That check is needed there for now, until the tables are separated.

Reply:
Sure!